### DISCUSSION TITLE
No module named 'typer'

### DISCUSSION THREAD

**Discussion by Ngsc-team** [AntonOsika/gpt-engineer#454] [Q&A]
this is the error msg:File "C:\Users\LBQ\gpt-engineer\gpt_engineer\main.py", line 7, in <module>
    import typer
ModuleNotFoundError: No module named 'typer'

I am using chatgpt's api key and change the line 22 to gpt-3.5.gpt 3.5-turbo,and gpt3.5-turbo-16k, (all fail）

also,there is no requirements.txt file exist,or generated in the codes.

I am using Anconda powershell prompt terminal.

**Comment by HarryAdney** [comment]
Hi, the requirements.txt file are no longer required, per update on 14th June.

HTH
